### PR TITLE
Fix replacement of service pictogram paths with absolute URLs in API

### PR DESCRIPTION
**Pull request by marcantoinedupre**
Also expand to 'description_teaser' and 'ambiance' fields.

Should address issue [#3321](https://github.com/GeotrekCE/Geotrek-admin/issues/3321)

### CODE REVIEW

File: geotrek/api/v2/serializers.py [hunk 1]
```diff
@@ -788,6 +767,28 @@ def get_departure_city(self, obj):
...
 
+        def _replace_image_paths_with_urls(self, data):
+            def replace(html_content):
+                if not html_content:
+                    return html_content
+                soup = BeautifulSoup(html_content, features="lxml")
+                imgs = soup.find_all('img')
+                for img in imgs:
+                    if img.attrs['src'][0] == '/':
+                        base_url = self.context.get("request").build_absolute_uri("/")[:-1]
```
Comment (submarcos) [inline geotrek/api/v2/serializers.py:778]: try to use directly build_absolute_url to get final url : self.context.get("request").build_absolute_uri(img.attrs['src'])

File: geotrek/api/v2/serializers.py [hunk 2]
```diff
@@ -788,6 +767,28 @@ def get_departure_city(self, obj):
...
+                        img['src'] = base_url + img.attrs["src"]
+                # Note: beautifulsoup makes a valid HTML doc from the data,
+                # but we don't want <html> or <body> in the output
+                return "".join([str(c) for c in soup.find("body").children])
```
Comment (submarcos) [inline geotrek/api/v2/serializers.py:782]: I think there is an option or method in beautifulsoup for this

File: geotrek/api/v2/serializers.py (hunk 1 above)
Comment (marcantoinedupre) [reply geotrek/api/v2/serializers.py]: Right, I have done it.

File: geotrek/api/v2/serializers.py (hunk 2 above)
Comment (marcantoinedupre) [reply geotrek/api/v2/serializers.py]: Right my complicated statement is useless. By passing `features="html.parser"` to BeautifulSoup constructor no extra HTML tags are added to the input (no <html> or <body>).

**Pull request merged**